library/ssl_tls12_server.c

```diff
@@ -3893,11 +3893,13 @@ static int ssl_parse_client_key_exchange(mbedtls_ssl_context *ssl)
             return MBEDTLS_ERR_SSL_DECODE_ERROR;
         }
 
+#if !defined(PSA_WANT_ALG_FFDH)
```

Comment:
Same.

Reply:
Tracked in https://github.com/Mbed-TLS/mbedtls/issues/7612#issuecomment-1568184319 so marking as resolved here.